test/integration/scheduler/queue_test.go

```diff
@@ -501,6 +521,25 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod1"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected by the PodAntiAffinity plugin is requeued when updating the label of the node to make it match the pod anti affinity",
+			initialNodes: []*v1.Node{st.MakeNode().Name("fake-node").Label("node", "fake-node").Obj()},
+			pods: []*v1.Pod{
+				// - pod1 and pod2 will be rejected by the PodAntiAffinity plugin.
+				st.MakePod().Name("pod1").PodAntiAffinityExists("bbb", "zone", st.PodAntiAffinityWithRequiredReq).Container("image").Obj(),
+				st.MakePod().Name("pod2").PodAntiAffinityExists("ccc", "region", st.PodAntiAffinityWithRequiredReq).Container("image").Obj(),
+			},
+
+			triggerFn: func(testCtx *testutils.TestContext) error {
+				// Add label to the node which will make it match pod1's podAntiAffinity.
+				if _, err := testCtx.ClientSet.CoreV1().Nodes().Update(testCtx.Ctx, st.MakeNode().Name("fake-node").Label("zone", "zone1").Obj(), metav1.UpdateOptions{}); err != nil {
+					return fmt.Errorf("failed to update pod1: %w", err)
+				}
+				return nil
+			},
+			wantRequeuedPods:          sets.New("pod1"),
```

Comment:
I found that the integration test will report an error. Can you help me find the reason? I am not sure if I understand it wrong. I did not find the reason for the error. @sanposhiho :) thanks

Reply:
Failed tests are:
- Pod_rejected_by_the_interPodAffinity_plugin_is_requeued_when_creating_the_label_of_the_node_to_make_it_match_the_pod_anti_affinity_
- Pod_rejected_by_the_interPodAffinity_plugin_is_requeued_when_updating_the_label_of_the_node_to_make_it_match_the_pod_anti_affinity_

Commented the failure reason above.